diagnosis/getkuberneteslogs.sh

```diff
@@ -191,6 +165,27 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
+#checks if azure-cli is installed   
+requirements
+
+#workaround for SSL interception
+export AZURE_CLI_DISABLE_CONNECTION_VERIFICATION=1 
+export ADAL_PYTHON_SSL_NO_VERIFY=1
+
+#Validate resource-group
+location=$(az group show -n $RESOURCE_GROUP --query location)
+if [ $? -ne 0 ]; then
+        echo "[$(date +%Y%m%d%H%M%S)][ERR] Specified Resource group not found."
+        exit 1
+fi
+
+#Get the master nodes from the resource group
+master_nodes=$(az resource list -g $RESOURCE_GROUP --resource-type "Microsoft.Compute/virtualMachines" --query "[?tags.poolName=='master'].{Name:name}" --output table)
+if [ $? -ne 0 ]; then
+        echo "[$(date +%Y%m%d%H%M%S)][ERR] Kubernetes master nodes not found in the resource group."
+        exit 1
+fi
+
 if [ -n "$MASTER_HOST" ]
```

Comment:
We can remove flag `MASTER_HOST` once we obtain the master nodes using azure-cli

Reply:
Will do that in next iteration

Reply:
I will get the master ip and then remove this flag in next iteration